.eslintrc.js

```diff
@@ -1,20 +1,60 @@
-/**@type {import('eslint').Linter.Config} */
-// eslint-disable-next-line no-undef
-module.exports = {
-  // root: true,
-  // parser: '@typescript-eslint/parser',
-  // plugins: [
-  // 	'@typescript-eslint',
-  // ],
-  // extends: [
-  // 	'eslint:recommended',
-  // 	'plugin:@typescript-eslint/recommended',
-  // ],
-  // rules: {
-  // 	'semi': [2, "always"],
-  // 	'@typescript-eslint/no-unused-vars': 0,
-  // 	'@typescript-eslint/no-explicit-any': 0,
-  // 	'@typescript-eslint/explicit-module-boundary-types': 0,
-  // 	'@typescript-eslint/no-non-null-assertion': 0,
-  // }
+/** @type {import('eslint').Linter.Config} */
+const config = {
```

Comment:
We define `config` as a variable so we can attach the `Linter.Config` type using jsdoc. This is supposed to work with `module.exports` but doesn't due to https://github.com/microsoft/TypeScript/issues/47107.